bigbluebutton-html5/private/config/settings.yml

```
@@ -967,7 +967,38 @@ public:
     enabled: true
     interval: 10000
     timeout: 10000
+    probes: 5 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)